### DISCUSSION TITLE
Use CommunityToolkit.Mvvm in wpf.ui library

### DISCUSSION THREAD

**Discussion by IvanDmitriev1** [lepoco/wpfui#323] [Ideas]
What about to use `CommunityToolkit.Mvvm` `RelayCommand` instead of those that already exist?

**Comment by pomianowski** [comment]
The Microsoft Toolkit has been replaced with the Community Toolkit in https://github.com/lepoco/wpfui/commit/bbaebb8332d8f46fcd5c12c4c1e84d4cc37ad566

As for `RelayCommand`, this is cannot be realized because **WPF UI** cannot depend on the Community Toolkit by default.  
The user may don't need to use additional libraries, so there should be as few of them as possible.
